docs/ocean/design_docs/prognostic-sediment-transport.rst

```diff
@@ -0,0 +1,415 @@
+
+  .. math::
+
+      C_{ref} = \frac{0.331(\theta - 0.045)^{1.75}}{1+0.72(\theta-0.045)^{1.75}}
+
+  and :math:`z_{ref}=2D_{50}`. Again, :math:`\theta` is the Shields parameter and
+  :math:`D_{50}` is the sediment median grain diameter.
+
+- `Goldstein et al., 2014 <https://esurf.copernicus.org/articles/2/67/2014/esurf-2-67-2014-discussion.html>`_
+
+  .. math::
+
+      C_{ref} = \left[ \frac{0.328U_b}{0.0688+1000D_{50}}\right]^2
+
+  at height :math:`z_{ref}=0.01 m`, where :math:`U_b` is the magnitude of the bottom velocity (:math:`m\,s^{-1}`)
+
+**Process-based approach**
+
+The proces-based approach solves SSC by the three-dimensional advection-diffusion equations,
+with an additional source/sink term for vertical settling and exchange with the bed
+
+.. math::
+
+      \frac{\partial{H(z)C(z)} }{\partial t} + \nabla \cdot \left( H(z)C(z) \bf{u}\right) = \nabla \cdot \left ( \epsilon_s \nabla H(z)C(z) \right) + C_{source}
+
+with the addtional term written as
+
+.. math::
+
+      C_{source} = -w_s C(z) + E_s
+
+in which :math:`H(z)` is the layer thickness (:math:`m`) in the ocean model,
+:math:`t` is time (:math:`s`), :math:`\nabla` represents the advection process,
+:math:`\bf{u}` is the velocity vector (:math:`m\,s^{-1}`),
+:math:`E_s` is the erosion source (:math:`kg\,m^{-2}\,s^{-1}`), which is computed as
+
+.. math::
+
+      E_s = E_0(1-\phi)\frac{\tau_{sf}-\tau_{ce}}{\tau_{ce}}
+
+where :math:`E_0` is the bed erodibility constant (:math:`kg\,m^{-2}\,s^{-1}`),
+:math:`\phi` is the porosity of the bed layer,
+:math:`\tau_{ce}` is the bed critical erosion stress (:math:`m^2\,s^{2}`)
+
+**Sediment settling velocity**
+
+Suspended sediment transport also requires the knowledge of the sediment settling velocity.
+Sediment settling velocity is a function of the sediment geometry (e.g., particle size, shape and roundness),
+the fluid characteristics (density and viscosity), and well as the fluid turbulence velocity.
+
+Two sediment settling veloity formulations are selected to implement into MPAS-Ocean, including
+
+- `Soulsby, 1997 <https://www.icevirtuallibrary.com/doi/abs/10.1680/doms.25844.fm>`_
+
+  .. math::
+
+      w_s = \frac{\nu}{D_{50}}\left[\sqrt{ (10.36^2+1.049D_*^3)}-10.36\right]
+
+  where :math:`\nu` is the kinematic viscosity of water (:math:`m^2\,s^{-1}`).
+
+- `Cheng, 1997 <https://doi.org/10.1061/(ASCE)0733-9429(1997)123:2(149)>`_
+
+  .. math::
+
+      w_s = \frac{\nu}{D_{50}} (\sqrt{25+1.2D_*^2}-5)^{1.5}
+
+  in which :math:`D_* = D_{50}\left[\frac{(s-1)g}{\nu ^2}\right]^{1/3}` is dimensionless grain size.
+
+
+
+Implementation
+--------------
+
+Implementation: Bedload sediment transport
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Date last modified: 2021/01/19
+
+Contributors: Zhendong Cao, Xylar Asay-Davis
+
+The sediment bedload transport requires the pre-definitions of the following namelist options:
+
+- A logical variable to switch ON/OFF the bedload transport: :code:`config_sediment_bedload_enabled`
```

Comment:
```suggestion
- A logical variable to switch ON/OFF the bedload transport: :code:`config_sediment_bedload_enable`
```
For consistency with some of the other config variables

Reply:
I changed all these, thanks. @caozd999 all the flags use `enable` at the end without the d

Reply:
@mark-petersen Many thanks Mark.